### PR TITLE
check iip without vmi

### PR DESCRIPTION
**Pull request by sureshkvl**

### CODE REVIEW

File: gremlin-checks/checks.groovy [hunk 1]
```diff
@@ -195,3 +195,8 @@ check("route target belonging to several tenants (only RT matching the pattern '
      )
      .filter{it.get().value("display_name").matches(rtPattern)}
 )
+
+check("instance-ip without virtual-machine-interface",
+    g.V().hasLabel('instance_ip')
+         .not(__.out().hasLabel("virtual_machine_interface"))
+)
```
Comment (eonpatapon) [inline gremlin-checks/checks.groovy:201]: Can you remove the "virtual-network with instance-ip but without any virtual-machine-interface" check as well?

**Review by eonpatapon — CHANGES_REQUESTED** [review]

File: gremlin-checks/checks.groovy (hunk 1 above)
Comment (eonpatapon) [reply gremlin-checks/checks.groovy:201]: In checks.py too

Comment (sureshkvl) [reply gremlin-checks/checks.groovy:201]: yes updated

File: gremlin-fsck/gremlin_fsck/checks.py [hunk 2]
```diff
@@ -13,18 +13,6 @@
...
-    """instance-ip without any virtual-machine-interface
-    """
-    return g.V().hasLabel("virtual_network").not_(
-        __.in_().hasLabel('virtual_machine_interface')
-    ).in_().hasLabel("instance_ip")
-
-
 @log_json
 @count_lines
 def clean_vn_with_iip_without_vmi(iips):
```
Comment (eonpatapon) [inline gremlin-fsck/gremlin_fsck/checks.py]: You need to update this clean method for your new check

**Review by eonpatapon — CHANGES_REQUESTED** [review]

**Review by eonpatapon — APPROVED** [review]

**Pull request merged**